tests/test_executors.py

```diff
@@ -5,9 +5,27 @@
 
 # %%
 from protein_search.backend.my_executors import *
+from jina import DocumentArray, Flow
+from jina.types.document.generators import from_csv
 
 # %%
 def test_ProtBERT_encoding():
     sequences = ["A E T C Z A O", "T E A B Z A O"]
     docs = DocumentArray((Document(content=seq) for seq in sequences))
     embedded_docs = ProtBertExecutor().encode(docs)
+
+# %%
+def test_Indexer_indexing_samelength():
+    proteins = DocumentArray(
+        from_csv(
+            fp=open("data/samelength.csv"),
+            field_resolver={"Protein sequences": "text"},
+        )
+    )
+
+    flow = Flow(port_expose=12345).add(uses=ProtBertExecutor).add(uses=MyIndexer)
+    with flow:
+        flow.index(proteins)
```

Comment:
The function `test_Indexer_indexing_samelength` is supposed to test the indexer, not the entire flow. We could instantiate `MyIndexer` directly like for the `ProtBertExecutor` test function. To reuse some code, we could do something like:

```suggestion
sequences = ["A E T C Z A O", "T E A B Z A O"]

def encode_sequences(sequences):
    docs = DocumentArray((Document(content=seq) for seq in sequences))
    embedded_docs = ProtBertExecutor().encode(docs)
    
    return embedded_docs

def test_ProtBERT_encoding():
    embedded_docs = encode_sequences(sequences)

# %%
def test_Indexer_indexing_samelength():
    embedded_docs = encode_sequences(sequences)
    
    # indexing
    indexed_docs = MyIndexer().index(embedded_docs)
```

Reply:
We might also add a third test to test the complete flow, but as this PR is about the indexer that could also wait.

Reply:
Good idea, however the MyIndexer requires self.workspace to be initialised as so index cannot be used as a static method. I'll raise an issue and we can implement a test in the future.

Reply:
I agree, let's open an issue and move on. I'll merge this.